### ISSUE TITLE
API technical renewable potential

### ISSUE THREAD

**Issue by Julie231113** [NREL/developer.nrel.gov#306]
Dear Mrs/Mr,
I tried to use the API to retrieve the renewable technical potential for a given location, however I have the feeling that the documentation https://developer.nrel.gov/docs/energy-optimization/rede-technical-potential/#endpoints is not pointing to the updated version of the server. Could you help me to make the API work ? 

Best,
Julie

**Comment by PjEdwards** [comment]
I'll need to research this. The endpoints are in theory still deployed as documented, however I ran a quick test and did receive a 502 Bad Gateway response. I'll see what I can find out.

**Comment by PjEdwards** [comment]
@Julie231113 The version of this technical potential analysis tool as documented has been updated in the RE Data Explorer, and these docs are no longer accurate. However, there is a far better tool available now for running your own technical potential analysis via code which is NREL's [reV Technical Potential analysis tool](https://www.nrel.gov/gis/renewable-energy-potential.html). reV installation and usage documentation can be found at [https://nrel.github.io/reV/index.html.](https://nrel.github.io/reV/index.html). This is in fact the library that now sites behind the RE Data Explorer UI based tool.

I've updated our docs to reflect this as well: https://developer.nrel.gov/docs/energy-optimization/rede-technical-potential/
